docs/src/install/core.rst

```diff
@@ -2,31 +2,34 @@
 Installing Oríon
 ****************
 
-Oríon should work on most Linux distributions and Mac OS X. It is tested on Ubuntu 16.04 LTS and Mac
-OS X 10.13. We do not support Windows and there is no short term plan to do so.
+Oríon is compatible with most Linux distributions and Mac OS X. It is tested on Ubuntu 16.04 LTS and
+Mac OS X 10.13. We do not support Windows and there is no short term plan to do so.
 
-Via PyPI
-========
-
-The easiest way to install Oríon is using the Python package manager. The core of Oríon is
-registered on PyPI under `orion`.
+The easiest way to install the latest version of Oríon is through the Python package manager. Oríon
```

Comment:
This is a bit confusing because `latest` in docs now points to develop branch while stable points to `master`, which is what you get using PyPI.

Reply:
Do you mean that on PyPI, `latest` means the last version to have been released? If it's the case we have to either roll with it as the `latest` on PyPI isn't related to our `latest` readthedoc channel or we have to remove the `stable` channel; we can't have 2 channels pointing to the same thing, it's even more confusing. 
Which alternative do you prefer?

Reply:
Develop is not available on PyPI, so yes latest would be the equivalent of master. We need to have access to develop version on readthedocs. Maybe latest and develop, or stable and develop?

Reply:
That's was I was thinking about too as a solution. I prefer having `stable` and `develop` because these channel names are more descriptive and `stable` channel is a staple in readthedocs ecosystem.